change/@ni-nimble-angular-a238e91b-0109-4e07-b89a-aba8b2d19387.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Moving comment to a discussion:

>> Should we add an example of the usage of this in the example project?
>
> I don't feel strongly. Wouldn't necessarily do it for every column, maybe 1-2 if done. Whichever way just a reminder to keep it consistent across with blazor

Reply:
If no one feels strongly, I lean towards not putting it in the example apps because I don't think it adds much value. There isn't anything framework-specific about this property, which is already exposed on all relevant storybook stories.

@atmgrifter00, was your question because you want this feature exposed in the example apps or because you just wanted to make sure we had considered it?

Reply:
Just that we had considered it. Don't feel strongly either.